src/Damage.js

```diff
@@ -51,7 +57,13 @@ class Damage {
    */
   evaluate(target) {
     let amount = this.amount;
+
     if (this.attacker) {
+      const critChance = Math.max(this.attacker.getMaxAttribute('critical') || 0, 0);
+      this.critical = Random.probability(critChance);
+      if (this.critical) {
+        amount = Math.ceil(amount * 1.5);
```

Comment:
I think this amount multiplier should be a configuration option on the Damage constructor that way the skill/effect/whatever can control the amount. Example: a fireball does 4x crit damage or whatever.